backend/organization/models/project.py

```diff
@@ -110,6 +113,25 @@ class Project(models.Model):
         blank=True
     )
 
+    # Field not in use. Keeping temporarily for backwards compatibility
+    location = models.CharField(
+        help_text="Project location",
+        verbose_name="Location",
+        max_length=2048,
+        null=True,
+        blank=True
+    )
```

Comment:
We can remove this field.

Reply:
or replace this with `loc` column.

Reply:
Yep, would remove this and rename loc to location